beacon-chain/blockchain/metrics.go

```diff
@@ -199,6 +199,20 @@ var (
 			Buckets: []float64{1, 5, 20, 100, 500, 1000},
 		},
 	)
+	reorgDistance = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "reorg_distance",
+			Help:    "Captures distance of reorgs",
+			Buckets: []float64{1, 2, 4, 8, 16, 32, 64},
+		},
+	)
+	reorgDepth = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "reorg_depth",
+			Help:    "Captures depth of reorgs",
```

Comment:
Could you please add some details to the help text? I.e. what does depth / distance mean?

Reply:
> Could you please add some details to the help text? I.e. what does depth / distance mean?

Yep. Done